src/registrar/templates/domain_detail.html

```diff
@@ -1,6 +1,33 @@
 {% extends "domain_base.html" %}
+{% load static url_helpers %}
 
 {% block domain_content %}
   {{ block.super }}
-  <p>Active: {% if domain.is_active %}Yes{% else %}No{% endif %}</p>
+  <div class="margin-top-4 tablet:grid-col-10">
+
+    {% url 'domain-nameservers' pk=domain.id as url %}
+    {% if domain.nameservers %} 
+      {% include "includes/summary_item.html" with title='DNS name servers' value=domain.nameservers list='true' edit_link=url %}
+    {% else %}
+      <h2 class="margin-top-neg-1"> DNS name servers </h2>
+      <p> No DNS name servers have been added yet. Before your domain can be used we’ll need information about your domain name servers.</p>
+      <a class="usa-button margin-bottom-1" href="{{url}}"> Add DNS name servers </a> 
+    {% endif %}
```

Comment:
I have a couple of thoughts on this code.

1. `domain.nameservers` will be an [n+1 query](https://stackoverflow.com/a/97253) to the registry (first to fetch the domain, then once for each name server), so we could improve the load time of this page by checking if `domain.is_active` before accessing `domain.nameservers`. (I may be able to improve this further in the domain interface PR I am working on by introducing a `domain.has_nameservers`.)
1. Even if the domain is active and has nameservers, given that it will slow the page load to display them, we might want to consider in user research whether this is information that users will find valuable on the summary page. (Speaking _only_ for myself, I rarely visit a registrar to reference what nameservers already exist unless I am also intending to update/change them.)
1. Some of the domain servers will have IP addresses. Probably not something to worry about on this PR, but just good to be aware of: `domain.nameservers` won't return a simple list of strings, once implemented, rather, it will return a complex data object.
1. Worth considering, if users do find nameservers valuable on this page, should we also include DNSSEC summary on this page?